CHANGELOG.md

```diff
@@ -10,6 +17,8 @@
 
 **Features**:
 
+- Add a CLI command `prqlc collect` to collect a project's modules into a single
+  file (@aljazerzen, #3739)
```

Comment:
I think it would be better to set rules for whether or not updates related to the CLI should be included here.
If I remember correctly, in the past few releases it was included in the Integrations section instead of the Features section.

Reply:
Great, done, thanks